src/Proxies/WP.php

```diff
@@ -177,4 +178,76 @@ public function number_format_i18n( float $number, int $decimals = 0 ): string {
+	 *
+	 * @param array|string $args     Optional. An array of key => value arguments to match against
+	 *                               the post type objects. Default empty array.
+	 * @param string       $output   Optional. The type of output to return. Accepts post type 'names'
+	 *                               or 'objects'. Default 'names'.
+	 * @param string       $operator Optional. The logical operation to perform. 'or' means only one
+	 *                               element from the array needs to match; 'and' means all elements
+	 *                               must match; 'not' means no elements may match. Default 'and'.
+	 * @return string[]|WP_Post_Type[] An array of post type names or objects.
+	 */
+	public function get_post_types( array $args = [], string $output = 'names', string $operator = 'and' ): array {
+		return get_post_types( $args, $output, $operator );
+	}
+
+	/**
+	 * Retrieves a list of registered taxonomy names or objects.
+	 *
+	 * @since x.x.x
+	 *
+	 * @param array  $args     Optional. An array of `key => value` arguments to match against the taxonomy objects.
+	 *                         Default empty array.
+	 * @param string $output   Optional. The type of output to return in the array. Accepts either taxonomy 'names'
+	 *                         or 'objects'. Default 'names'.
+	 * @param string $operator Optional. The logical operation to perform. Accepts 'and' or 'or'. 'or' means only
+	 *                         one element from the array needs to match; 'and' means all elements must match.
+	 *                         Default 'and'.
+	 * @return string[]|WP_Taxonomy[] An array of taxonomy names or objects.
+	 */
+	public function get_taxonomies( array $args = [], string $output = 'names', string $operator = 'and' ): array {
+		return get_taxonomies( $args, $output, $operator );
+	}
+
+	/**
+	 * Retrieves the terms in a given taxonomy or list of taxonomies.
+	 *
+	 * @since x.x.x
+	 *
+	 * @param array|string $args       Optional. Array or string of arguments. See WP_Term_Query::__construct()
+	 *                                 for information on accepted arguments. Default empty array.
+	 * @param array|string $deprecated Optional. Argument array, when using the legacy function parameter format.
+	 *                                 If present, this parameter will be interpreted as `$args`, and the first
+	 *                                 function parameter will be parsed as a taxonomy or array of taxonomies.
+	 *                                 Default empty.
+	 * @return WP_Term[]|int[]|string[]|string|WP_Error Array of terms, a count thereof as a numeric string,
+	 *                                                  or WP_Error if any of the taxonomies do not exist.
+	 *                                                  See the function description for more information.
+	 */
+	public function get_terms( $args = [], $deprecated = '' ) {
+		return get_terms( $args, $deprecated );
+	}
 }
```

Comment:
Do we really need to Proxy these functions?

Reply:
Maybe is for testing them easier???

Reply:
Yes, it makes it easier to mock those functionalities.